### PR TITLE
Adds the Mk-43

### PR DESCRIPTION
**Pull request by LadyFowls**
Remaking the pr due to a issue involving sprites and fucking something up.

Adds the MK-43 chambered in a chambered down 5.56 varient called the 5.43 to be used inside for security operations.

### CODE REVIEW

**Comment by LadyFowls** [comment]
!review

File: code/game/machinery/autolathe_datums.dm [hunk 1]
```diff
@@ -290,6 +290,17 @@
 	path = /obj/item/ammo_magazine/c38/rubber
 	category = "Arms and Ammunition"
 
+/datum/autolathe/recipe/magazine_543
+	name = "ammunition (5.43mm)"
```
Comment (NonQueueingMatt) [inline code/game/machinery/autolathe_datums.dm]: This is a very snowflake ammo caliber. Why not just nerf 5.56 slightly, buff 7.62 and use 556?

File: code/modules/projectiles/ammunition/boxes.dm [hunk 2]
```diff
@@ -207,6 +207,23 @@
 	origin_tech = list(TECH_COMBAT = 3)
 	ammo_type = /obj/item/ammo_casing/a556/ap
 
+/obj/item/ammo_magazine/mk43
+	name = "magazine (5.43mm) "
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/ammunition/boxes.dm]:
```suggestion
	name = "magazine (5.43mm)"
```

File: code/modules/projectiles/ammunition/boxes.dm [hunk 3]
```diff
@@ -207,6 +207,23 @@
...
+	origin_tech = list(TECH_COMBAT = 4)
+	mag_type = MAGAZINE
+	caliber = "5mm"
+	matter = list(DEFAULT_WALL_MATERIAL = 1800)
+	ammo_type = /obj/item/ammo_casing/mk43
+	max_ammo = 10
+	multiple_sprites = 1
+
+/obj/item/ammo_magazine/mk43/rubber
+	name = "magazine (5.43mm Rubber) "
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/ammunition/boxes.dm]:
```suggestion
	name = "magazine (5.43mm rubber)"
```

File: code/modules/projectiles/guns/projectile/rifle.dm [hunk 4]
```diff
@@ -250,6 +250,43 @@
 	update_held_icon()
 	return
 
+
+/obj/item/gun/projectile/automatic/rifle/mk43
+	name = "Mark 43 service rifle"
+	desc = "A semi-automatic rifle chambered in a slimmed down 5.56 varient, used for internal operations involving less penetration"
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/guns/projectile/rifle.dm]:
```suggestion
	desc = "A semi-automatic rifle chambered in a slimmed down 5.56 variant, used for internal operations in areas where heavier calibers are not needed."
```

File: code/modules/projectiles/guns/projectile/rifle.dm [hunk 5]
```diff
@@ -250,6 +250,43 @@
 	update_held_icon()
 	return
 
+
+/obj/item/gun/projectile/automatic/rifle/mk43
+	name = "Mark 43 service rifle"
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/guns/projectile/rifle.dm]: No specific names in the name. Use a generic name here and put the actual name in the desc.

File: html/changelogs/isdrifle.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds the MK 43 ISD Service Rifle, chambered in a necked down 5.43 varient for indoor operations."
```
Comment (NonQueueingMatt) [inline html/changelogs/isdrifle.yml]:
```suggestion
  - rscadd: "Adds the MK 43 ISD Service Rifle, chambered in a necked down 5.43 variant for indoor operations."
```

**Comment by NonQueueingMatt** [comment]
I also think this should be added to the armoury in the initial implementation.

File: code/game/machinery/autolathe_datums.dm [hunk 7]
```diff
@@ -290,6 +290,17 @@
 	path = /obj/item/ammo_magazine/c38/rubber
 	category = "Arms and Ammunition"
 
+/datum/autolathe/recipe/magazine_556short
+	name = "ammunition (5.56mm Ten Round)"
```
Comment (NonQueueingMatt) [inline code/game/machinery/autolathe_datums.dm]: what's a "ten round" supposed to be?

File: code/modules/projectiles/ammunition/boxes.dm [hunk 8]
```diff
@@ -207,6 +207,23 @@
 	origin_tech = list(TECH_COMBAT = 3)
 	ammo_type = /obj/item/ammo_casing/a556/ap
 
+/obj/item/ammo_magazine/a556/mk43
+	name = "magazine (5.56mm) Ten Round "
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/ammunition/boxes.dm]:
```suggestion
	name = "magazine (5.56mmR)"
```

File: code/modules/projectiles/ammunition/boxes.dm [hunk 9]
```diff
@@ -207,6 +207,23 @@
...
+	origin_tech = list(TECH_COMBAT = 4)
+	mag_type = MAGAZINE
+	caliber = "a556"
+	matter = list(DEFAULT_WALL_MATERIAL = 1800)
+	ammo_type = /obj/item/ammo_casing/a556/mk43
+	max_ammo = 10
+	multiple_sprites = 1
+
+/obj/item/ammo_magazine/a556/mk43/rubber
+	name = "magazine (5.56mm Rubber) Ten Round "
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/ammunition/boxes.dm]:
```suggestion
	name = "magazine (5.56mmR rubber)"
```

File: code/modules/projectiles/ammunition/bullets.dm [hunk 10]
```diff
@@ -217,6 +217,18 @@
 	desc = "A 5.56mm practice bullet casing."
 	projectile_type = /obj/item/projectile/bullet/rifle/a556/practice
 
+
+/obj/item/ammo_casing/a556/mk43
+	desc = "A 5.56mm round."
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/ammunition/bullets.dm]:
```suggestion
	desc = "A 5.56mmR round."
```

File: code/modules/projectiles/guns/projectile/rifle.dm [hunk 11]
```diff
@@ -250,6 +250,43 @@
...
+	ammo_type = /obj/item/ammo_casing/a556/mk43
+	magazine_type = /obj/item/ammo_magazine/a556/mk43/rubber
+	allowed_magazines = list(/obj/item/ammo_magazine/a556/mk43)
+	accuracy = 2
+	recoil_wielded = 1
+	accuracy_wielded = 3
+	multi_aim = 0
+
+	firemodes = list(
+		list(mode_name="semiauto", burst=1, fire_delay= 10, move_delay=null, burst_accuracy=null, dispersion=null)
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/guns/projectile/rifle.dm]: That's a long ass firing delay. Is this intentional?

File: html/changelogs/isdrifle.yml [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds the MK 43 ISD Service Rifle, chambered in a necked down 5.56 varient for indoor operations."
+  - tweak: "Swaps the 7.62 and 556 values"
```
Comment (NonQueueingMatt) [inline html/changelogs/isdrifle.yml]: This doesn't mean anything to the average person. Say what it actually does.

File: html/changelogs/isdrifle.yml [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds the MK 43 ISD Service Rifle, chambered in a necked down 5.56 varient for indoor operations."
```
Comment (NonQueueingMatt) [inline html/changelogs/isdrifle.yml]:
```suggestion
  - rscadd: "Adds the MK 43 Service Rifle, chambered in a slimmed down 5.56 variant for indoor operations."
```
It's not a security rifle if it doesn't exist in security.

File: code/modules/projectiles/projectile/bullets.dm [hunk 14]
```diff
@@ -184,10 +184,21 @@
...
+	damage = 25
+
+/obj/item/projectile/bullet/rifle/mk43
+	damage = 20
+
+/obj/item/projectile/bullet/rifle/mk43/rubber
+	name = "rubber bullet"
+	check_armour = "melee"
+	damage = 6
+	agony = 50
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/projectile/bullets.dm]: I'd make this a bit higher. Fifty agony is like, not that much. Just ten above a .45 rubber bullet.

**Comment by NonQueueingMatt** [comment]
Conflicts everywhere, inactive author. Reopen when ready.

**Pull request closed without merging**